assets/js/modules/analytics/datastore/properties.js

```diff
@@ -117,13 +116,17 @@ export const actions = {
 			};
 			const { properties, profiles, matchedProperty } = response;
 			const { dispatch } = yield Data.commonActions.getRegistry();
-			yield actions.receiveProperties( properties );
-			dispatch( STORE_NAME ).receiveProfiles( profiles );
+			yield actions.receiveProperties( properties, { accountID } );
 
 			if ( matchedProperty ) {
 				yield actions.receiveMatchedProperty( matchedProperty );
 			}
 
+			if ( properties.length && properties[ 0 ] && properties[ 0 ].id ) {
```

Comment:
See other comment, I think this would be more accurate relying on `profiles[ 0 ].webPropertyId` instead.

Reply:
As above for `accounts.js` I'm happy to make the change if that is the correct thing to do :)

Reply:
Yep, that is more accurate, the current logic will likely result in errors as soon as you have more than one property.

Reply:
Fixed